<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editting_form</class>
 <widget class="QWidget" name="editting_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QSpinBox" name="spinBox_4"/>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="spinBox_3"/>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="spinBox_2"/>
   </item>
   <item row="0" column="0">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
